lapce-core/src/indent.rs

```diff
@@ -11,18 +11,19 @@ pub enum IndentStyle {
     Spaces(u8),
 }
 
+pub const DEFAULT_INDENT: IndentStyle = IndentStyle::Spaces(4);
```

Comment:
This would be better as a Default implementation or a constant of `IndentStyle`

Reply:
Will do.